### ISSUE TITLE
Filtering by feature frequency does not apply filter

### ISSUE THREAD

**Issue by adswafford** [biocore/qiita#2685]

**Comment by antgonza** [comment]
Can you add a job id?

**Comment by adswafford** [comment]
https://qiita.ucsd.edu/analysis/description/18757/
Job 0ad4cb6c-14a0-4c58-afe8-9364b603827a

https://qiita.ucsd.edu/analysis/description/18756/
Job 108716bf-afba-4b65-847a-83e426fb8c86

https://qiita.ucsd.edu/analysis/description/18755/
Job bec39a22-9fc2-43c5-af6d-cafeb36a8031
Job 348ec646-4018-4e00-8ab2-b744cf3153cf

https://qiita.ucsd.edu/analysis/description/18754/
Job 8795b156-bcfb-4fce-88f0-b75b7be2435e
Job 9d575c83-50ba-4200-8a52-03a337dc0933

**Comment by adswafford** [comment]
I set Minimum feature frequency across samples to 10 but the feature tables still have features with frequency less than 10. I also thought it might be because I left the WHERE clause empty for some of them, but I tried again with that field also filled in and the low abundance features still weren't removed.

**Comment by antgonza** [comment]
I just open the first example (https://qiita.ucsd.edu/analysis/description/18757/, Job 0ad4cb6c-14a0-4c58-afe8-9364b603827a) and IMOO it worked as expected: removed any features with less than the value asked for: 10. See screenshots of summaries:
![summary](https://user-images.githubusercontent.com/2014559/46089186-346cb100-c16b-11e8-8194-cd6546cf4e81.png)

**Comment by adswafford** [comment]
Those stats don't seem to reflect what's in the table based on the feature details tab:
![image](https://user-images.githubusercontent.com/27017606/46093277-c5438c80-c16b-11e8-8748-cb9daf660ac2.png)

**Comment by adswafford** [comment]
Confirmed with @antgonza , needs more troubleshooting to determine cause

**Comment by antgonza** [comment]
@adswafford, could you check if in this is still happening in the new qp-qiime2 release? Right now you can test in the qiita-rc system.

**Comment by shaekin** [comment]
Unfortunately it doesn't look like the problem is fixed.  Here's a study I tried to do this with:
https://qiita.ucsd.edu/analysis/description/18757/
The filtered table ID is ID: 63994
I did a minimum feature frequency across samples of 11 (it wouldn't let me do 10 since that was exactly the same as a previous job that succeeded).  Here's a screen shot of the feature detail:
![image](https://user-images.githubusercontent.com/31516412/48573784-3547d680-e8c2-11e8-876f-c368be1bbd17.png)
There are some sevens, sixes, and a four in there, and that's just the top of the list.

**Comment by shaekin** [comment]
Sorry!  I realized immediately after posting that response that I was not testing in Qiita rc.  In Qiita rc, I'm coming up with an error when I try to run the filter process.  Here's the analysis I'm trying to run it on: 
https://qiita-rc.ucsd.edu/analysis/description/18133/
Heres the job I tried to run: Job ae98085e-6e0c-43ca-82fb-6b110477be90
I ran the job twice, but it failed both times.  The error message says "ValueError: `ids_to_keep` must contain at least one ID."  This is when I'm trying to run the Filter Features from tables command.  This command doesn't actually have an option for which ids to keep.
